### ISSUE TITLE
cannot use hydra's allow_termination_from support without X-Forwarded-Proto header

### ISSUE THREAD

**Issue by amihalj** [ory/hydra-maester#32]
I am trying to use TLS termination according to hydra server spec:
[TLS termination](https://www.ory.sh/docs/hydra/production#tls-termination)

but there is not way to send X-Forwarded-Proto header in maester requests.

Is it possible to include property in helm values that allows such header to be sent?

thank you.

(Eg. hydra admin log)
```

level=info msg="started handling request" method=POST remote="172.17.0.6:41622" request=/clients
time="2019-11-06T08:51:53Z" level=warning msg="Could not serve http connection" error="X-Forwarded-Proto header is missing"
time="2019-11-06T08:51:53Z" level=error msg="An error occurred while handling a request" code=502 debug= details="map[]" error="can not serve request over insecure http"
```

**Comment by aeneasr** [comment]
I believe a simple flag like `--forwarded-proto=https` would be enough?

/cc @jakkab @piotrmsc

**Comment by aeneasr** [comment]
This is resolved now with #35
